front/pages/w/[wId]/builder/data-sources/public-urls.tsx

```diff
@@ -213,12 +214,12 @@ export default function DataSourcesView({
                 );
               }}
               subElement={
-                <div className="flex items-center gap-1 pl-2 pt-0.5 text-xs text-element-700">
-                  <span>Added by: {ds.editedByUser?.fullName}</span>
-                  <span className="px-1 text-element-500">|</span>
-                  <span>{dataSourcesUsage[ds.id] ?? 0}</span>
-                  <RobotStrokeIcon />
-                </div>
+                <>
+                  Added by: {ds.editedByUser?.fullName}
+                  <span className="mx-2 h-3 w-0.5 bg-element-500" />
+                  Used by: {dataSourcesUsage[ds.id] ?? 0}
```

Comment:
`nit`: I would probably wrap text in `<span>` tags even though text doesn't require styling